### DISCUSSION TITLE
How to set the default width and height

### DISCUSSION THREAD

**Discussion by sczh01** [beeware/toga#2875] [Debugging help]
### What is the problem or limitation you are having?

I try the mulitinputlinetext example, and found some limitations

default layout:
![image](https://github.com/user-attachments/assets/e335ef39-f123-4cf0-ba36-aae57e688d1b)

I want it's below layout, when I create these UI elements that can config some setting.

![image](https://github.com/user-attachments/assets/cc7691ec-acd5-4464-ada8-2c5dd5a0cc93)

### Describe the solution you'd like

Maybe the Pack class can do it, but I can't find the points. I used the 'width', but it will fixed the width size when I zoom the windows size.

![image](https://github.com/user-attachments/assets/be3cacbc-d6c9-4d53-9766-c2cb489bfe9f)

### Describe alternatives you've considered

Just for above information now.

**Comment by freakboy3742** [comment]
Converted to a discussion so that we can determine what (if any) action is needed.

Unfortunately, it's not clear to me what you see as the problem here. You've provided 2 screen shots, and other than the fact that the windows are different sizes (and thus so are the widgets), I can't see any substantial difference.

If the issue is that the buttons are being "clipped" - that will be because your layout is constraining the available size. You *don't" want to use width, because that will only apply more constraints; it's likely related to the constraints you've implicitly applied by having a split container with a defined ratio. 

If the issue is the width of the Table's columns - that looks like a bug. There's no ability to control the width of columns; but they should be auto-filling to use all the available space. This is something that we *do* test for; so there's clearly something specific about your layout that isn't *just* about the Table. If you're able to provide a more complete example of your layout, we can investigate further.
